src/components/settings-menu/UserDevSection.js

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback } from 'react';
+import { InteractionManager } from 'react-native';
+import { ScrollView, Switch } from 'react-native-gesture-handler';
+import { useDispatch } from 'react-redux';
+import networkTypes from '../../helpers/networkTypes';
+import { settingsUpdateNetwork } from '../../redux/settings';
+import { Column } from '../layout';
+import { ListFooter, ListItem } from '../list';
+import NetworkSection from './NetworkSection';
+
+import {
```

Comment:
can u cleanup the spacing here - remove spacing above and insert a space below this import line